### PR TITLE
Add "manage.py queue_depth" command

### PR DESCRIPTION
**Pull request by j4mie**
This adds a command `manage.py queue_depth` (takes an optional argument for the queue name) and just prints the current depth of the queue. Intended to be run periodically and used for monitoring.

### CODE REVIEW

File: django_dbq/management/commands/queue_depth.py [hunk 1]
```diff
@@ -0,0 +1,15 @@
+from django.core.management.base import BaseCommand
+from django_dbq.models import Job
+
+
+class Command(BaseCommand):
+
+    help = "Print the current depth of the given queue"
+
+    def add_arguments(self, parser):
+        parser.add_argument('queue_name', nargs='?', default='default', type=str)
```
Comment (RealOrangeOne) [inline django_dbq/management/commands/queue_depth.py]: Why is the number of args here undefined? This'll allow multiple queue names, which may do weird things to the logging and querying

Comment (j4mie) [reply django_dbq/management/commands/queue_depth.py]: I just copied it from the `worker` command! If I do `python manage.py foo bar` it says `error: unrecognized arguments: bar`

**Comment by j4mie** [comment]
Note to self: this should include an optional `--output-format` arg, which allows the user to specify they they want the log output message to look like. Eg: `--output-format "queue_depth={queue_depth}"`

**Comment by jordaneb** [comment]
Implemented in #33

**Pull request closed without merging**